pages/index.js

```diff
@@ -47,18 +48,20 @@ export default function Home() {
       <Navmenu />
       <Valuefields
         addvalue={addvalue}
-        handlesubtractvalues={handleSubtractValues}
-        onChangeSubtract={handleSubtractValue}
+        minusvalue={minusvalue}
+        handleSubtractValues={handleSubtractValues1}
+        onChangeSubtract={handleSubtractValues}
         onChangeValue={handleAddValue}
```

Comment:
Naming: You shouldn't have functions `handleSubtractValues` and `handleSubtractValues1`. It's not clear what's the difference between these functions.